ui_framework/src/components/popover/_popover.scss

```diff
@@ -1,7 +1,8 @@
 // Pop menu is an animated popover relatively positioned to a button / action.
 // By default it positions in the middle, but can be anchored left and right.
 
-@include component('kuiPopMenu') {
+@include component('kuiPopover') {
```

Comment:
The changes with how you're doing positioning here break the usage.

![](https://d3vv6lp55qjaqc.cloudfront.net/items/3r430i3n0O1y3k3w2S3S/Image%202017-07-25%20at%2011.16.16%20AM.png?X-CloudApp-Visitor-Id=59773&v=c137283f)